jobsub/examples/gbl_local/config.cfg

```diff
@@ -51,10 +51,10 @@ HotpixelRunNumber	= @RunNumber@
 SkipNEvents		= 0
 
 # Output subfolder structure
-DatabasePath		= ./output/database
-HistogramPath		= ./output/histograms
-LcioPath            = ./output/lcio
-LogPath				= ./output/logs
+DatabasePath		= %(BasePath)s/output/database
```

Comment:
I'm not quite sure, whether I get you right.

The thing is: For me, it does not work with `./output/xxx`. Did I forget setting some environment variables?

Since this is an example, I figured to better deliver something, which is more restrictive, but therefore working out of the box. If the mess up is on my side, I am happy to revert this.